src/environments/.gitignore

```diff
@@ -0,0 +1,5 @@
+environment.*.ts
+
+!environment.prod.ts
```

Comment:
This should say `!environment.production.ts`, as that's the name of the required file in the codebase.

Reply:
Ah yes. Good catch.